controllers/ArticleController.js

```diff
@@ -1,6 +1,7 @@
+import { Op } from 'sequelize';
 import cloudinary from '../config/cloudinary';
 import Utilities from '../helpers/utilities';
-import { Article, User } from '../models';
+import { Article, User, Like } from '../models';
```

Comment:
Where are you using `Like` @OKiMaureen

Reply:
@andela-aonifade, it is used in the article controller. If you view the code, you will see being used.